src/fromager/wheels.py

```diff
@@ -189,6 +175,22 @@ def add_extra_metadata_to_wheels(
                 f"{req.name}: {wheel_file} does not contain {dist_info_dir.name}"
             )
 
+        if extra_data_plugin:
+            data_to_add = overrides.invoke(
```

Comment:
A while back, @tiran added plugin references pointing to the default functions for all of our overrides so that consumers downstream can find those functions and check their function signatures against the customizations in the downstream build. I missed that when we added this new override without a default. We don't have to hold up this PR over it, but we should have an issue ticket to make sure we add that.